src/components/layout.js

```diff
@@ -1,40 +1,41 @@
 import React from "react";
-import styled from "styled-components";
 import Head from "next/head";
 import Header from "./general/Header";
-import "./../reset.css";
 import Footer from "./general/Footer";
-import { ThemeProvider } from "styled-components";
-import theme from "./../theme";
+import { Container, Typography } from "@material-ui/core";
+import { makeStyles, ThemeProvider } from "@material-ui/core/styles";
+import theme from "../theme";
+
+const useStyles = makeStyles(theme => ({
+  mainHeading: {
+    textAlign: "center",
+    margin: `${theme.spacing(4)}px 0`
+  }
+}));
 
 export default function Layout({ title, children }) {
+  const classes = useStyles();
+
   return (
     <>
       <Head>
         <title>{title}</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <ThemeProvider theme={theme}>
-        <LayoutContainer>
+        <Container maxWidth="lg">
```

Comment:
Why did you limit the width of this container? I feel like it makes sense if the header and footer stretch over the whole page.